week2/project/ecommerce/src/buttonSet.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Suggestion:
```suggestion
import { useState, useEffect } from 'react';
```
you don't need tp  'import React' for modern React (17+), same as other `import React from 'react';`
[Reference](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#:~:text=With%20the%20new%20transform%2C%20you%20can%20use%20JSX%20without%20importing%20React.)

Reply:
[fixed in 149e398](https://github.com/HackYourAssignment/React-Cohort50/pull/12/commits/149e398c9b607c5bee8f004cba9b697381813362)